content/chapters/compute/lab/README.md

```diff
@@ -282,6 +282,201 @@ Therefore, there is also a gray area where there is no clear right answer to thi
+www-data      93       1     112  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data      93       1     117  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data      93       1     118  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data      93       1     119  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data      93       1     120  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data      99       1      99  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data      99       1     128  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data      99       1     129  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data      99       1     134  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data      99       1     135  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data      99       1     136  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data      99       1     137  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     101       1     101  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     101       1     138  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     101       1     139  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     101       1     140  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     101       1     141  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     101       1     142  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     101       1     143  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     105       1     105  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     105       1     126  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     105       1     127  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     105       1     130  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     105       1     131  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     105       1     132  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     105       1     133  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     109       1     109  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     109       1     115  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     109       1     116  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     109       1     121  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     109       1     122  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     109       1     123  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+www-data     109       1     124  0    7 21:07 pts/0    00:00:00 httpd -DFOREGROUND
+root         146       0     146  0    1 21:10 pts/1    00:00:00 bash
+root         152     146     152  0    1 21:10 pts/1    00:00:00 ps -efL
```

Comment:
There are too many lines here. Either trim them (simply delete a few lines in the middle and replace them with a line like this: `[...]`) or, better, lower the thresholds in the configs and the number of connections created by `make_conn.py`. It's very easy to get lost in such a lengthy output.